libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue

```diff
@@ -152,6 +152,7 @@ const {
 } = useDropdown({
   clickawayIgnoreSelector: `div[data-toggle-id="${dataToggleId}"]`,
   offset: { mainAxis: 0, crossAxis: 0 },
+  layoutShift: false,
```

Comment:
@abemscac I noticed some dropdown menus will repeatedly receive the `autoUpdate` callback from `@floating-ui`. Once the menu is clicked, it will settle and only be called as expected. Setting `autoUpdate` option `layoutShift` to false fixes this, although it means the menus won't automatically shift to the top/bottom on scroll.

For now this is ok, since I don't really like the automatic menu shifting anyway, but it would be nice to figure out the autoUpdate issue. It might be related to the flex layout interaction with `IntersectionObserver` in floating-ui, but I didn't figure out the exact reason.